experiences/facebook/facebook.test.js

```diff
@@ -67,28 +68,28 @@ const facebookTest = function() {
 		expect(isAtActiveSnapPoint).equal(true);
 	});
 
-	it('remove card, should not be displayed in viewport', async function() {
-		await removeCard(
-			browser,
-			config.cards.facebook.cardSelector
-		);
-
-		await waitUntil(browser, async()=>{
-			const firstCard = await browser.$(config.cards.facebook.cardSelector);
-
-			const firstCardIsInViewport = await firstCard.isDisplayedInViewport();
-
-			return firstCardIsInViewport;
-		}, false, {
-			interval: 1000,
-			timeoutMsg: 'Card was still in viewport after removing it.'
-		}, async()=>{
-			await removeCard(
-				browser,
-				config.cards.facebook.cardSelector
-			);
-		});
-	});
+	// it('remove card, should not be displayed in viewport', async function() {
+	// 	await removeCard(
+	// 		browser,
+	// 		config.cards.facebook.cardSelector
+	// 	);
+
+	// 	await waitUntil(browser, async()=>{
+	// 		const firstCard = await browser.$(config.cards.facebook.cardSelector);
+
+	// 		const firstCardIsInViewport = await firstCard.isDisplayedInViewport();
+
+	// 		return firstCardIsInViewport;
+	// 	}, false, {
+	// 		interval: 1000,
+	// 		timeoutMsg: 'Card was still in viewport after removing it.'
+	// 	}, async()=>{
+	// 		await removeCard(
+	// 			browser,
+	// 			config.cards.facebook.cardSelector
+	// 		);
+	// 	});
+	// });
```

Comment:
Is this intended or you commited by mistake?

Reply:
intended

Reply:
if intended better to go `it.skip` as that will be part of jest output, rather than commented code for which we lose track

Reply:
> if intended better to go `it.skip` as that will be part of jest output, rather than commented code for which we lose track

Agree on this.

There's a [coding-standard repo with Jest rules](https://github.com/Marfeel/coding-standards/tree/master/packages/eslint-config-jest), not sure if it should be applied here since it's using mocha (and chai?) instead. 

I guess we need to make sure it is skipped intentionally to avoid developers skip all the tests to bypass CI and that's it.